org.eclipse.jdt.ui.tests/ui/org/eclipse/jdt/ui/tests/quickfix/CleanUpTest1d5.java

```diff
@@ -4246,4 +4267,73 @@ public void testConvertStringBufferToStringBuilderAll() throws Exception {
 				new HashSet<>(Arrays.asList(MultiFixMessages.StringBufferToStringBuilderCleanUp_description)));
 	}
 
+	@Test
+	public void testOverrideMultiFix() throws Exception {
+		Hashtable<String, String> opts= JavaCore.getOptions();
+		String oldOverrideError = opts.get(JavaCore.COMPILER_PB_MISSING_OVERRIDE_ANNOTATION);
```

Comment:
Is this necessary ? I know you want to reset to the old value so as to not disturb other tests, but according to https://github.com/eclipse-jdt/eclipse.jdt.ui/blob/344ae32efe1d334d37635899ec2455a6aa665685/org.eclipse.jdt.ui.tests/ui/org/eclipse/jdt/ui/tests/quickfix/CleanUpTestCase.java#L95 that should already happen. This class does inherit from there.

Reply:
Good catch!